alpha/apps/kaltura/lib/media/myFlvStreamer.class.php

```diff
@@ -61,9 +56,11 @@ public function streamFlv()
 	}
 
 	// TODO - should move to some metadata wrapper
-	public static function getAllAssetsIds ( $filePath )
+	public static function getAllAssetsIds ( $entry )
 	{
-		list ( $xml_doc , $xpath ) = self::getDomAndXpath ( $filePath );
+		$filePath = self::addRootPath($entry->getDataPath());
+		$xmlStr = file_get_contents($filePath);
```

Comment:
@david-winder-kaltura  
Any place you touch file_get_contents , use similar method from kFile::file_get_contents ().
Also add it to kFile.